macro/run_simScript.py

```diff
@@ -27,10 +26,7 @@
 
 # Default dark photon parameters
 theDPmass    = 0.2*u.GeV
-
-# Alpaca
-motherMode = True
-
```

Comment:
Please rebase your branch against master: this undoes a change by @ThomasRuf .

You should remove the commits included in the old pull request from this rebase to ensure everything works well (e.g. drop them in the interactive rebase).